pkg/controller/ads/ads_processor.go

```diff
@@ -52,11 +52,15 @@ type processor struct {
 	ack       *service_discovery_v3.DiscoveryRequest
 	req       *service_discovery_v3.DiscoveryRequest
 	lastNonce *lastNonce
+	// the channel used to send domains to dns resolver. key is domain name and value is refreshrate
+	DnsResolverChan chan []*config_cluster_v3.Cluster
 }
 
 func newProcessor() *processor {
 	return &processor{
 		Cache:     NewAdsCache(),
+		ack:       nil,
+		req:       nil,
```

Comment:
nit: unneccessary

Reply:
those are not added by me, should be deleted in another PR